README.md

```diff
@@ -1,105 +1,103 @@
-
-## Running Your Tests
-
-Run your test suite using the command:
-
-```bash
-npm test
-```
-
-The test results will output to the terminal.
-
----
-
-## Linting Your Code
-
-Run the command in your terminal `npm run lint` to run the linter on your JavaScript code. There will be errors and warnings right from the start in this starter kit - the linter is still running successfully.
-
-Your linter will look at the JavaScript files you have within the `src` directory and the `test` directory. 
-
-## Webpack?
-
-If you look in the `package.json` file, you'll see one of the library dependencies called `webpack`. If you're interested in learning more about what Webpack is and how it works behind the scenes, take a look through the [Webpack configuration documentation](https://webpack.js.org/concepts/).
-
-## Deploying to GitHub Pages
-
-_If you are finished with the functionality and testing of your project_, then you can consider deploying your project to the web! This way anyone can play it without cloning down your repo.
-
-[GitHub Pages](https://pages.github.com/) is a great way to deploy your project to the web. Don't worry about this until your project is free of bugs and well tested!
-
-If you _are_ done, you can follow [this procedure](./gh-pages-procedure.md) to get your project live on GitHub Pages.
+[![Contributors][contributors-shield]](https://github.com/Atos20/what-s-cooking-refactor/graphs/contributors)
+[![Forks][forks-shield]](https://github.com/Atos20/what-s-cooking-refactor/network/members)
+[![Issues][issues-shield]](https://github.com/Atos20/what-s-cooking-refactor/issues)
+
+## Foodamental _*group project*_
+
+## Contributors
+  - [Kyle Jones](https://github.com/KJJones13)
+  - [Blake Donn](https://github.com/BlakeDonn)
+  - [Orlando Murcio](https://www.github.com/atos20)
+
+## Context
+
+### Set up
+* On the top right corner of this page, click the **Fork** button.
+- Clone the repository to your computer `git clone <URL>`
+  - When you run git clone - git clone [remote-address] [what you want to name the repo]
+  replace the [...] with the terminal command arguments): `git clone [remote-address] [what you want to name the repo]`
+- `cd` into the repository with the following command `cd <repo-name>`
+- Run `npm install`
+- Run `open src/index.html`
+
+### Contribute
+
+- Create a new branch with `git checkout -b <new branch name>`
+- Open your text editor and add or remove functionalities to the site.
+- `git add` and `git commit -m "<your commit meessage>"` to save the changes to your local repository
+- `git push` your changes
+- Create a new pull request!
+
+#### Technologies used:
+  * JavaScript ES6
+  * Terminal
+  * GitHub
+  * HTML
+  * SCSS
+  * Dependencies
+
+## Challenges
+
+    - API's
+    - manipulation of nested data types
+    - use of chai-spies to spy on DOM related methods
+
+## Wins
```

Comment:
Added some of  my winds but please feel free to add some of yours, we could potentially change the title to `team wins`